src/sidebar/test/annotation-metadata-test.js

```diff
@@ -247,6 +247,35 @@ describe('annotation-metadata', function () {
     });
   });
 
+  describe('.isPublic', function () {
+    it('returns true if an annotation is shared within a group', function () {
+      assert.isTrue(annotationMetadata.isPublic(fixtures.publicAnnotation()));
+    });
+
+    it('returns false if an annotation is not publicly readable', function () {
+      var annotation = Object.assign(fixtures.defaultAnnotation(), {
+        permissions: {
+          read:['acct:someemail@localhost'],
+        },
+      });
+      assert.isFalse(annotationMetadata.isPublic(annotation));
+    });
+
+    it('returns null if an annotation is missing permissions', function () {
+      var annotation = Object.assign(fixtures.defaultAnnotation());
+      assert.isFalse(annotationMetadata.isPublic(annotation));
+    });
+
+    it('returns false if an annotation is not publicly readable', function () {
```

Comment:
This description is exactly the same as the one above. This suggests you might want to use a data-driven test (with the `unroll` helper). This can happen after merging.